odata_query/grammar.py

```diff
@@ -63,40 +64,43 @@
 
 class ODataLexer(Lexer):
```

Comment:
Looking at `sly`s source code, strings seem to work just as well and don't make `mypy` explode: